Sources/ClientGtk/Storage/StorageManager.swift

```diff
@@ -0,0 +1,89 @@
+import DeltaCore
+import Foundation
+
+final class StorageManager {
+	static var `default` = StorageManager()
```

Comment:
This whole first section of the class is indented with tabs instead of spaces

Reply:
Oh, whoops